nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/BaseHealthCheck.java

```diff
@@ -140,6 +144,10 @@ public boolean isReady() {
         O options = getOptions();
         return !isInProgress() && options != null && options.isEnabled();
     }
+    
+    public boolean isEnabled(){     
+        return getOptions().isEnabled();
```

Comment:
pretty sure this can cause NPE. See `ìsReady()`. Also maybe `isReady()` can use the `isEnabled()` in its logic.

Reply:
`getOptions().isEnabled()` is a `boolean`, so it can't be `null`. The `isReady()` method already uses that `isEnabled()` method.

Reply:
options can be `null`